.babelrc.js

```diff
@@ -4,9 +4,6 @@ module.exports = {
       presets: [
         ['@babel/preset-env', { targets: { node: 'current' } }],
       ],
-      plugins: [
-        'require-context-hook'
-      ],
```

Comment:
The environment variable NODE_ENV=test essentially caused the static storybook to be build with this babel config.